core/src/cg/group4/Launcher.java

```diff
@@ -73,10 +88,30 @@ public final void create() {
         cTimeKeeper = TimerStore.getInstance().getTimeKeeper();
 
         cStandUp = StandUp.getInstance();
+        
+        int fa = 0, fb = 0, fc = 0;
```

Comment:
Temporary test code, do not forget to remove when all the features are attached.